examples/http-disable-chunking/http_disable_chunking.bal

```diff
@@ -13,14 +13,14 @@ service chunkingSample on new http:Listener(9092) {
     @http:ResourceConfig {
         path: "/"
     }
-    //Parameters include a reference to the caller endpoint and an object with the request data.
+    //Parameters include a reference to the caller endpoint and an object containing the request data.
```

Comment:
I think we can get rid of this line altogether. This is an unnecessary comment, unrelated to the context